models/cow_protocol/ethereum/cow_protocol_ethereum_schema.yml

```diff
@@ -171,3 +171,29 @@ models:
       - &token_approvals
         name: token_approvals
         description: Number of ERC20 token approvals made within the settlement
+  - name: cow_protocol_ethereum_app_data
+    meta:
+      blockchain: ethereum
+      project: cow_protocol
+      contributors: bh2smith
+    config:
+      tags: ['ethereum','cow_protocol','app_data', "metadata"]
+    description: >
+      CoW Protocol App Data is JSON content stored on IPFS corresponding to order AppHash (cf. CoW Documentation https://docs.cow.fi/front-end/creating-app-ids).
```

Comment:
Its all here! I can't understand why this keeps failing the pre-commit hooks.

Reply:
i think it's because your model file name omitted `ethereum` in it, but it has it in the schema file. i pushed a change

Reply:
Silly me! I can't believe I missed that.